src/server/session.ts

```diff
@@ -2089,7 +2089,12 @@ export class Session<TMessage = string> implements EventSender {
         projects = filter(projects, p => p.languageServiceEnabled && !p.isOrphan());
         if (!ignoreNoProjectError && (!projects || !projects.length) && !symLinkedProjects) {
             this.projectService.logErrorForScriptInfoNotFound(args.file ?? args.projectFileName);
-            return Errors.ThrowNoProject();
+            if (some(projects, p => !p.languageServiceEnabled)) {
+                Errors.ThrowProjectLanguageServiceDisabled();

```

Comment:
Can you show case where this happens. I dont think we ever check language service disabled when getting project